homework/index.js

```diff
@@ -30,15 +32,123 @@
     return elem;
   }
 
-  function main(url) {
-    fetchJSON(url, (err, data) => {
-      const root = document.getElementById('root');
-      if (err) {
-        createAndAppend('div', root, { text: err.message, class: 'alert-error' });
-      } else {
-        createAndAppend('pre', root, { text: JSON.stringify(data, null, 2) });
+  function render(repositories, root) {
+    function leftContainer(index) {
+      const div = createAndAppend('div', root, { class: 'leftContainer' });
+      const pName = createAndAppend('p', div, {
+        text: 'Repository name: \n',
+        class: 'details',
+        id: 'bold',
+      });
+      createAndAppend('a', pName, {
+        text: repositories[index].name,
+        class: 'details',
+        href: repositories[index].html_url,
+        target: '_blank',
+      });
+
+      const pDescription = createAndAppend('p', div, {
+        text: '\n Description: ',
+        class: 'details',
+        id: 'bold',
+      });
+      createAndAppend('p', pDescription, {
+        text: repositories[index].description,
+        class: 'details',
+      });
+
+      const pForks = createAndAppend('p', div, {
+        text: '\n Forks: ',
+        class: 'details',
+        id: 'bold',
+      });
+      createAndAppend('p', pForks, {
+        text: repositories[index].forks_count,
+        class: 'details',
+      });
+
+      const pCreate = createAndAppend('p', div, {
+        text: '\n Created: ',
+        class: 'details',
+        id: 'bold',
```

Comment:
This generates invalid HTML: `id` attribute values must be unique. You already used the id `bold` in line 63.